datahub/dsr.py

```diff
@@ -1,36 +1,31 @@
 """This module defines the data structures for the MEDUSA Demand Simulator model."""
 from typing import Any
 
+import numpy as np
 from pydantic import BaseModel, Field
 
 
 class DSRModel(BaseModel):
     """Define required key values for Demand Side Response data."""
 
-    amount: list[float] = Field(alias="Amount", size=(13,))
-    cost: list[list[float]] = Field(alias="Cost", size=(1440, 13))
-    kwh_cost: list[float] = Field(alias="kWh Cost", size=(2,))
-    activities: list[list[float]] = Field(alias="Activities", size=(1440, 7))
-    activities_outside_home: list[list[float]] = Field(
+    amount: list = Field(alias="Amount", size=(13,))
+    cost: list = Field(alias="Cost", size=(1440, 13))
+    kwh_cost: list = Field(alias="kWh Cost", size=(2,))
+    activities: list = Field(alias="Activities", size=(1440, 7))
+    activities_outside_home: list = Field(
```

Comment:
I'd stick a comment before this block saying why your types aren't more specific.